service/application/src/main/java/com/solace/maas/ep/event/management/agent/logging/StreamingAppender.java

```diff
@@ -28,21 +28,53 @@ public class StreamingAppender extends AppenderBase<ILoggingEvent> {
     protected void append(ILoggingEvent event) {
         if (!standalone) {
             if (StringUtils.isNotEmpty(event.getMDCPropertyMap().get(RouteConstants.SCAN_ID))) {
-                sendLogsAsync(event,
+                sendScanLogsAsync(event,
                         event.getMDCPropertyMap().get(RouteConstants.SCAN_ID),
                         event.getMDCPropertyMap().get(RouteConstants.TRACE_ID),
                         event.getMDCPropertyMap().get(RouteConstants.ACTOR_ID),
                         event.getMDCPropertyMap().get(RouteConstants.SCAN_TYPE),
                         event.getMDCPropertyMap().get(RouteConstants.SCHEDULE_ID),
```

Comment:
The schedule Id was something we added for scans, I don't think we'll need it for configpush.

Reply:
done.